src/modeles/etape.js

```diff
@@ -0,0 +1,14 @@
+const InformationsHomologation = require('./informationsHomologation');
+
+class Etape extends InformationsHomologation {
+  constructor(donnees, referentiel) {
+    super(donnees);
+    this.referentiel = referentiel;
+  }
+
+  estComplete() {
+    return !!this;
```

Comment:
Vu ensemble : c'est le linter qui nous force à utiliser `this`… mais ici on sait ce qu'on veut faire et ce serait `return false;`.
Je suis pour relaxer le linter ici.